bokehjs/src/lib/models/annotations/title.ts

```diff
@@ -22,7 +20,7 @@ export class TitleView extends TextAnnotationView {
     let sx: number, sy: number
     const {bbox} = this.layout
 
-    switch (this.panel.side) {
+    switch (this.panel!.side) {
```

Comment:
what is needed to get rid of the `!` ?

Reply:
This will eventually go away. Previously these models had `declare panel: SidePanel` to remove the `undefined` case inherited from a base class. Due to JS/TS technicalities around overriding attributes with getters/setters, I had to remove those declarations, which results in having to use `!` operator.